src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -125,56 +127,58 @@ enum CORINFO_InstructionSet
     InstructionSet_FMA=13,
     InstructionSet_LZCNT=14,
     InstructionSet_PCLMULQDQ=15,
-    InstructionSet_POPCNT=16,
-    InstructionSet_Vector128=17,
-    InstructionSet_Vector256=18,
-    InstructionSet_Vector512=19,
-    InstructionSet_AVXVNNI=20,
-    InstructionSet_MOVBE=21,
-    InstructionSet_X86Serialize=22,
-    InstructionSet_EVEX=23,
-    InstructionSet_AVX512F=24,
-    InstructionSet_AVX512F_VL=25,
-    InstructionSet_AVX512BW=26,
-    InstructionSet_AVX512BW_VL=27,
-    InstructionSet_AVX512CD=28,
-    InstructionSet_AVX512CD_VL=29,
-    InstructionSet_AVX512DQ=30,
-    InstructionSet_AVX512DQ_VL=31,
-    InstructionSet_AVX512VBMI=32,
-    InstructionSet_AVX512VBMI_VL=33,
-    InstructionSet_AVX10v1=34,
-    InstructionSet_AVX10v1_V512=35,
-    InstructionSet_VectorT128=36,
-    InstructionSet_VectorT256=37,
-    InstructionSet_VectorT512=38,
-    InstructionSet_X86Base_X64=39,
-    InstructionSet_SSE_X64=40,
-    InstructionSet_SSE2_X64=41,
-    InstructionSet_SSE3_X64=42,
-    InstructionSet_SSSE3_X64=43,
-    InstructionSet_SSE41_X64=44,
-    InstructionSet_SSE42_X64=45,
-    InstructionSet_AVX_X64=46,
-    InstructionSet_AVX2_X64=47,
-    InstructionSet_AES_X64=48,
-    InstructionSet_BMI1_X64=49,
-    InstructionSet_BMI2_X64=50,
-    InstructionSet_FMA_X64=51,
-    InstructionSet_LZCNT_X64=52,
-    InstructionSet_PCLMULQDQ_X64=53,
-    InstructionSet_POPCNT_X64=54,
-    InstructionSet_AVXVNNI_X64=55,
-    InstructionSet_MOVBE_X64=56,
-    InstructionSet_X86Serialize_X64=57,
-    InstructionSet_EVEX_X64=58,
-    InstructionSet_AVX512F_X64=59,
-    InstructionSet_AVX512BW_X64=60,
-    InstructionSet_AVX512CD_X64=61,
-    InstructionSet_AVX512DQ_X64=62,
-    InstructionSet_AVX512VBMI_X64=63,
-    InstructionSet_AVX10v1_X64=64,
-    InstructionSet_AVX10v1_V512_X64=65,
+    InstructionSet_VPCLMULQDQ=16,
+    InstructionSet_VPCLMULQDQ_V512=17,
```

Comment:
nit: It matches a bit better if we do `PCLMULQDQ_V256` and `PCLMULQDQ_V512`

Reply:
Yeah, that does look better